islandora_xacml_editor.module

```diff
@@ -292,15 +292,16 @@ function islandora_xacml_editor_after_build($form, &$form_state) {
  */
 function islandora_xacml_editor_manage_xacml_form(array $form, array &$form_state, AbstractObject $object) {
   module_load_include('inc', 'islandora', 'includes/utilities');
-  $dt = islandora_basic_collection_get_member_objects($object, 0, 20, 'view', 'islandora:collectionCModel');
-  $child_collections_pids = array();
+  // Hard code the XACML pager element because it needs to be unique.
```

Comment:
Copy-paste from https://github.com/Islandora/islandora_solution_pack_collection/commit/994c6df3d63609882bb9de9e7df1ab4aebb358d1#diff-2d69368f59ba81d5635cd8274dd66090R83 can we throw it into a helper?